linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny/oval/ubuntu.xml

```diff
@@ -8,15 +8,27 @@
              system is properly configured, it must appear only once in auth section. -->
         <criterion test_ref="test_accounts_passwords_pam_faillock_deny_common_pam_unix_auth"
             comment="pam_unix.so appears only once in auth section of common-auth"/>
-        <criterion test_ref="test_accounts_passwords_pam_faillock_deny_common_pam_faillock_auth"
-            comment="pam_faillock.so is properly defined in auth section of common-auth"/>
+        <criteria operator="AND" comment="Check if pam_faillock.so within common-auth is properly enabled">
```

Comment:
please revert all changes in this file.
we need to ensure order of items, which by breaking the check you don't anymore.
also disa stig is missing the preauth entry, that is required, therefore current oval is already covering it.